docs/tech/glossary.md

```diff
@@ -2,6 +2,12 @@
 
 Attestation { #attestation }
 : A data proof provided to the [State Connector](#state_connector) by a decentralized network of Attestation Providers that confirms the validity or otherwise of any request.
+Each attestation within a round is added to a merkle trie from which the resulting root hash is the attestation proof for the round.
```

Comment:
Worth defining "Merkle trie" in the Glossary too (I know this can be an endless rabbit hole, but this one seems important enough).
Also, shouldn't Merkle be capitalized?